### PR TITLE
Fix latestReleaseTag() unit test

### LINKED ISSUE #704: Test_getLatestReleaseTage fails on Travis
It seems to occur quite frequently
```
--- FAIL: Test_getLatestReleaseTag (1.36s)
	notify_test.go:18: error getting latest release TagName via API
```

### PR DESCRIPTION
**Pull request by mik-dass**
fixes #704

### CODE REVIEW

**Comment by cdrage** [comment]
So.... we're exceeding GitHub's API limit:


```sh
	notify_test.go:18: error getting latest release TagName via API, error: GET https://api.github.com/repos/redhat-developer/odo/releases/latest: 403 API rate limit exceeded for 35.184.96.71. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) [rate reset in 19m23s]
```

Because we're using the API for the test being written... We need to re-work this test.

@ashetty1

File: pkg/notify/notify_test.go [hunk 1]
```diff
@@ -31,8 +31,8 @@ func Test_getLatestReleaseTag(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			releaseTag, _ := getLatestReleaseTag()
-			if test.success == true {
+			releaseTag, err := getLatestReleaseTag()
```
Comment (anmolbabu) [inline pkg/notify/notify_test.go]: Can we mock this func instead using something like monkeypatching golang library and avoid genuine api call altogether? More info @ https://github.com/bouk/monkey

Why an actual get call in UT ?  I feel if you can mock the expected output tht would be more than sufficient in my opinion

**Comment by surajnarwade** [comment]
@syamgk thanks a lot , this LGTM :tada:

**Comment by cdrage** [comment]
This LGTM!

**Pull request merged**